packages/react/src/codemods/emotion-to-compiled/__tests__/emotion-to-compiled.test.tsx

```diff
@@ -377,6 +377,54 @@ describe('emotion-to-compiled transformer', () => {
     'it adds TODO comment for imports which are not resolved'
   );
 
+  defineInlineTest(
+    { default: transformer, parser: 'tsx' },
+    {},
+    `
+    // @top-level comment
+
+    /** @jsx jsx */
+    import { ClassNames, CSSObject, css as c, jsx } from '@emotion/core';
+    // comment 1
+    import * as React from 'react';
+    `,
+    `
+    /* TODO: (from codemod) "ClassNames" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    /* TODO: (from codemod) "CSSObject" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    // @top-level comment
+
+    import '@compiled/react';
+
+    // comment 1
+    import * as React from 'react';
+    `,
+    'it should not remove top level comments when transformed'
+  );
+
+  defineInlineTest(
+    { default: transformer, parser: 'tsx' },
+    {},
+    `
+    // @top-level comment
+
+    /** @jsx jsx */
+    import * as React from 'react';
+    // comment 1
+    import { ClassNames, CSSObject, css as c, jsx } from '@emotion/core';
+    `,
+    `
+    /* TODO: (from codemod) "ClassNames" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    /* TODO: (from codemod) "CSSObject" is not exported from "@compiled/react" at the moment. Please find an alternative for it. */
+    // @top-level comment
```

Comment:
the top level comment should stay as the top level comment - here its been moved down a few places

Reply:
hmm..its todo from codemod. User has to remove them as they are breaking changes. Problem is that its difficult to differentiate between top-level comment and comment just before imports:

```ts
// @top-level

// first comment 1
import * as React from 'react';
// first comment 2
import * as React2 from 'react2';
```

in above case its difficult to differentiate between `// @top-level` and `// first comment 1` 😞

Reply:
Hmm yeah ok, lgtm